cspell.yml

```
@@ -1,4 +1,6 @@
 language: en
+dictionaries:
+  - 'fullstack'
```

nit: but you don't need quotes in yaml for simple strings like this
```suggestion
  - fullstack
```